dialogue-core/src/main/java/com/palantir/dialogue/core/TraceEnrichingChannel.java

```diff
@@ -21,19 +21,32 @@
 import com.palantir.dialogue.Endpoint;
 import com.palantir.dialogue.Request;
 import com.palantir.dialogue.Response;
+import com.palantir.tracing.CloseableSpan;
+import com.palantir.tracing.DetachedSpan;
 import com.palantir.tracing.Tracer;
+import com.palantir.tracing.api.SpanType;
 import com.palantir.tracing.api.TraceHttpHeaders;
 
 /** A channel that adds Zipkin compatible tracing headers. */
 final class TraceEnrichingChannel implements Channel {
+    public static final String OPERATION = "Dialogue-http-request";
```

Comment:
Can this be private? The public modifier can be removed either way.

Reply:
thanks intellij